db/migrate/20240427023227_create_employee_teams.rb

```diff
@@ -3,6 +3,8 @@ def change
     create_table :employee_teams do |t|
       t.date :start_date
       t.date :end_date
+      t.references :employee, null: false
```

Comment:
These two lines are actually not needed, because these two things are added in a later migration: 20240430003654_add_employee_and_team_to_employee_teams.rb